ironfish/src/testUtilities/nodeTest.ts

```diff
@@ -80,6 +80,7 @@ export class NodeTest {
     sdk.config.setOverride('enableTelemetry', false)
     sdk.config.setOverride('enableAssetVerification', false)
     sdk.config.setOverride('confirmations', 0)
+    sdk.config.setOverride('enableWallet', true)
```

Comment:
Is this necessary since the default should be `true`?

Reply:
my bad, removed